### PR TITLE
remove weigthing options and rename to consistent PLS

### PR DESCRIPTION
**Pull request by juliuspfadt**
fixes https://github.com/jasp-stats/jasp-issues/issues/2356
fixes https://github.com/jasp-stats/jasp-issues/issues/2265

Suggestions of @FloSchuberth:
- set the handling of flipped signs always to "none"
- move ignore structural model checkbox

### CODE REVIEW

File: R/plssem.R [hunk 1]
```diff
@@ -19,6 +19,9 @@ PLSSEMInternal <- function(jaspResults, dataset, options, ...) {
   jaspResults$addCitation("Rademaker ME, Schuberth F (2020). cSEM: Composite-Based Structural Equation Modeling. Package version: 0.4.0, https://m-e-rademaker.github.io/cSEM/.")
 
 
+  sink(file="~/Downloads/log.txt")
+  on.exit(sink(NULL))
```
Comment (Kucharssim) [inline R/plssem.R:23]: Can you remove this? 😄

**Comment by juliuspfadt** [comment]
stupid of me. But to be honest, this is not yet really ready, adding the construct scores has been a bit of a mystery of constant recalculations, and I need to solve that first. Let me convert to draft to make it clear. Sorry about that

**Comment by FloSchuberth** [comment]
@juliuspfadt Hey Julius, in case you have questions about the calculation of the construct scores in PLS, we can also have a brief meeting...

**Comment by juliuspfadt** [comment]
Thanks :) But the issue is not getting the scores, but how they are added to the data set. So far, adding the columns to the data results in recalculation of the analysis, re-adding the columns, recalculation, .... infinity. :D

**Comment by JorisGoosen** [comment]
So brunos fix is in, a new nightly will be build soon enough that should stop the restarting

**Comment by juliuspfadt** [comment]
Thats great news!!! Thanks.

**Comment by juliuspfadt** [comment]
@Kucharssim, should I ask someone else to review?

**Comment by juliuspfadt** [comment]
@FloSchuberth any chance you have some spare time to test this? If you have, and need help installing the module, let me know :)

**Comment by FloSchuberth** [comment]
Hey @Julius ***@***.***>
Yes, I can have look.

And yes, a quick refresh would be nice.

What I have done so far:
I have created a fork of your branch: https://github.com/jasp-stats/jaspSem/tree/master
Then I used Rstudio to create a local copy of that fork: File->New project->Version control-> Git using my repository url: https://github.com/FloSchuberth/jaspSem

I have installed JASP version 19.01. I started it -> Preferences -> Advanced
There I checked Developer mode and have selected the local jaspSEM folder.

If I click on Install Module, I get an error, see file attached. Any ideas?

If it helps we can also schedule a brief meeting for next week.

Best regards,
Flo

**Comment by juliuspfadt** [comment]
I checked things with @FloSchuberth (expert from U-Twente) regarding the functionality, maybe someone can do a quick code check? @vandenman, or @FBartos?

**Review by Kucharssim — APPROVED** [review]
The code changes look reasonable to me!

**Comment by juliuspfadt** [comment]
Thanks :)

**Pull request merged**